zipkin-lens/src/components/GlobalSearch/ConditionTraceId.js

```diff
@@ -0,0 +1,65 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const maxCharacters = 16;
+
+const propTypes = {
+  value: PropTypes.string.isRequired,
+  onConditionChange: PropTypes.func.isRequired,
+  setNextFocusRef: PropTypes.func.isRequired,
+  onFocus: PropTypes.func.isRequired,
+  onBlur: PropTypes.func.isRequired,
+  isFocused: PropTypes.bool.isRequired,
+};
+
+class ConditionTraceId extends React.Component {
+  constructor(props) {
+    super(props);
+    this.inputRef = undefined;
+    this.handleKeyDown = this.handleKeyDown.bind(this);
+  }
+
+  handleKeyDown(e) {
+    if (e.keyCode === 13) {
```

Comment:
KeyCode is deprecated. We must use the `e.key` instead. More better to follow the instruction here everywhere we handle keyboard events. 
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode#Example

Reply:
LGTM other than that. Cheers.

Reply:
Thank you for information 🙇 
I only use `event.key` for now.